src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -3111,17 +3111,47 @@ paths:
         - security server
       summary: upload a configuration anchor file
       operationId: uploadAnchor
```

Comment:
Changed to PUT. In the Security Server init phase I think that POST should be the correct method. But it is possible that it will get it's own endpoint or something 🤷‍♂️